worldedit-bukkit/build.gradle.kts

```diff
@@ -22,10 +22,10 @@ configurations.all {
 dependencies {
     "api"(project(":worldedit-core"))
     "api"(project(":worldedit-libs:bukkit"))
-    "api"("org.bukkit:bukkit:1.13.2-R0.1-SNAPSHOT") {
+    "api"("org.bukkit:bukkit:1.15.2-R0.1-SNAPSHOT") {
```

Comment:
I wonder if there's something we can do to ensure that we don't use APIs that will break 1.13.2 compat